test/SILOptimizer/devirtualize2.sil

```diff
@@ -113,7 +113,7 @@ sil_vtable Bar {
 }
 
 sil_vtable Foo {
-  #Foo.ping!1: _TFC4main3Foo4pingfS0_FT_T_
+  #Bar.ping!1: _TFC4main3Foo4pingfS0_FT_T_
```

Comment:
@swiftix Can you double-check this? I think the old SIL was incorrect because Foo.ping was an override of Bar.ping in the AST, but not in the SIL vtable. So we devirtualized a call to Bar.ping() on an instance of Foo to Bar.ping and not Foo.ping as you would expect if Foo.ping was an override of Bar.ping.